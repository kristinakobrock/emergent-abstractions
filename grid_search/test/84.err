  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 20.05it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:02, 20.48it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 20.83it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 22.17it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 22.82it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 22.71it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.16it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 22.03it/s] 54%|#################################################6                                          | 27/50 [00:01<00:01, 22.81it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 26.85it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 25.16it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 23.89it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 25.03it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 25.98it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 26.52it/s]100%|############################################################################################| 50/50 [00:02<00:00, 29.70it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.76it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 20.22it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 20.79it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 23.71it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 22.79it/s]100%|############################################################################################| 13/13 [00:00<00:00, 22.78it/s]
