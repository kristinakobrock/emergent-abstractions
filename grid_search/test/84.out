# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6748457551002502, "acc": 0.6133410930633545, "length": 3.5347259044647217, "mode": "train", "epoch": 1}
{"loss": 0.6720300912857056, "acc": 0.6077483892440796, "length": 1.8359375, "mode": "test", "epoch": 1}
{"loss": 0.6674904227256775, "acc": 0.6200578212738037, "length": 3.5146684646606445, "mode": "train", "epoch": 2}
{"loss": 0.6613482236862183, "acc": 0.6238372325897217, "length": 2.55078125, "mode": "test", "epoch": 2}
{"loss": 0.6607393026351929, "acc": 0.6237333416938782, "length": 3.4563279151916504, "mode": "train", "epoch": 3}
{"loss": 0.6476235389709473, "acc": 0.6331267952919006, "length": 3.7109375, "mode": "test", "epoch": 3}
{"loss": 0.6542946696281433, "acc": 0.6263952851295471, "length": 3.509176731109619, "mode": "train", "epoch": 4}
{"loss": 0.6375772356987, "acc": 0.6422423720359802, "length": 3.97265625, "mode": "test", "epoch": 4}
{"loss": 0.6252674460411072, "acc": 0.6515239477157593, "length": 3.7166717052459717, "mode": "train", "epoch": 5}
{"loss": 0.6032694578170776, "acc": 0.6882811784744263, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5707109570503235, "acc": 0.7053700089454651, "length": 3.8779349327087402, "mode": "train", "epoch": 6}
{"loss": 0.5867435932159424, "acc": 0.6904296875, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5304337739944458, "acc": 0.7318470478057861, "length": 3.9442789554595947, "mode": "train", "epoch": 7}
{"loss": 0.5729571580886841, "acc": 0.7195311784744263, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5127453207969666, "acc": 0.7554454207420349, "length": 3.970282554626465, "mode": "train", "epoch": 8}
{"loss": 0.5626266598701477, "acc": 0.7259765267372131, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.49756377935409546, "acc": 0.7641497254371643, "length": 3.9791746139526367, "mode": "train", "epoch": 9}
{"loss": 0.5679063200950623, "acc": 0.7183594107627869, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.4851502478122711, "acc": 0.7666105628013611, "length": 3.984226942062378, "mode": "train", "epoch": 10}
{"loss": 0.5709776878356934, "acc": 0.7185546159744263, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.4807153642177582, "acc": 0.7757839560508728, "length": 3.988717794418335, "mode": "train", "epoch": 11}
{"loss": 0.576541543006897, "acc": 0.7257812023162842, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.46663978695869446, "acc": 0.7788355350494385, "length": 3.9922444820404053, "mode": "train", "epoch": 12}
{"loss": 0.5645689368247986, "acc": 0.7373046278953552, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.458204060792923, "acc": 0.7825496196746826, "length": 3.994014263153076, "mode": "train", "epoch": 13}
{"loss": 0.5577494502067566, "acc": 0.7333983778953552, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.4591546654701233, "acc": 0.7857128977775574, "length": 3.9955337047576904, "mode": "train", "epoch": 14}
{"loss": 0.5380764007568359, "acc": 0.7529297471046448, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.44554266333580017, "acc": 0.7908446788787842, "length": 3.9960715770721436, "mode": "train", "epoch": 15}
{"loss": 0.5248119235038757, "acc": 0.7568359375, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4398232102394104, "acc": 0.7938899993896484, "length": 3.9956445693969727, "mode": "train", "epoch": 16}
{"loss": 0.5130560398101807, "acc": 0.7623047232627869, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.432955801486969, "acc": 0.7979370951652527, "length": 3.9961013793945312, "mode": "train", "epoch": 17}
{"loss": 0.5047217607498169, "acc": 0.7681640982627869, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.43364039063453674, "acc": 0.802715003490448, "length": 3.9975924491882324, "mode": "train", "epoch": 18}
{"loss": 0.4993577003479004, "acc": 0.7699218392372131, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.43522119522094727, "acc": 0.8032550811767578, "length": 3.9977035522460938, "mode": "train", "epoch": 19}
{"loss": 0.49818360805511475, "acc": 0.7701172232627869, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.41024038195610046, "acc": 0.8120204210281372, "length": 3.9975452423095703, "mode": "train", "epoch": 20}
{"loss": 0.4744437336921692, "acc": 0.7880858778953552, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.40262269973754883, "acc": 0.8187401294708252, "length": 3.997537136077881, "mode": "train", "epoch": 21}
{"loss": 0.4620526134967804, "acc": 0.7943359017372131, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.39718350768089294, "acc": 0.8239448666572571, "length": 3.999009370803833, "mode": "train", "epoch": 22}
{"loss": 0.4723234474658966, "acc": 0.7876953482627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3929910361766815, "acc": 0.8301382660865784, "length": 3.9988203048706055, "mode": "train", "epoch": 23}
{"loss": 0.44839614629745483, "acc": 0.8095703125, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.38676998019218445, "acc": 0.8312556743621826, "length": 3.9982354640960693, "mode": "train", "epoch": 24}
{"loss": 0.4517839550971985, "acc": 0.8099609017372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.3720201849937439, "acc": 0.8409337997436523, "length": 3.9994802474975586, "mode": "train", "epoch": 25}
{"loss": 0.4544413089752197, "acc": 0.8085938096046448, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.37040287256240845, "acc": 0.842068612575531, "length": 3.9996118545532227, "mode": "train", "epoch": 26}
{"loss": 0.4209167957305908, "acc": 0.8185547590255737, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3616196811199188, "acc": 0.8482669591903687, "length": 3.999626636505127, "mode": "train", "epoch": 27}
{"loss": 0.4364858567714691, "acc": 0.8230467438697815, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3661632835865021, "acc": 0.8495776653289795, "length": 3.9996347427368164, "mode": "train", "epoch": 28}
{"loss": 0.45511671900749207, "acc": 0.8220701813697815, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.3612711727619171, "acc": 0.8495733141899109, "length": 3.999511480331421, "mode": "train", "epoch": 29}
{"loss": 0.40896928310394287, "acc": 0.8265624642372131, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.35510560870170593, "acc": 0.8513692617416382, "length": 3.9997053146362305, "mode": "train", "epoch": 30}
{"loss": 0.41572827100753784, "acc": 0.8357421159744263, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3542344868183136, "acc": 0.8541101813316345, "length": 3.9998841285705566, "mode": "train", "epoch": 31}
{"loss": 0.4010680019855499, "acc": 0.8400391340255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3554537892341614, "acc": 0.8564345240592957, "length": 3.999851703643799, "mode": "train", "epoch": 32}
{"loss": 0.4086008071899414, "acc": 0.830078125, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3427978456020355, "acc": 0.858460545539856, "length": 3.9998865127563477, "mode": "train", "epoch": 33}
{"loss": 0.4149569869041443, "acc": 0.8337889909744263, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.33980605006217957, "acc": 0.862751841545105, "length": 3.9998838901519775, "mode": "train", "epoch": 34}
{"loss": 0.40554821491241455, "acc": 0.8318358063697815, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3336825370788574, "acc": 0.8618587255477905, "length": 3.9999594688415527, "mode": "train", "epoch": 35}
{"loss": 0.40161043405532837, "acc": 0.8404297232627869, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3336374759674072, "acc": 0.8622710704803467, "length": 3.9994778633117676, "mode": "train", "epoch": 36}
{"loss": 0.4019118547439575, "acc": 0.8349609375, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3300318121910095, "acc": 0.8614413142204285, "length": 3.999948263168335, "mode": "train", "epoch": 37}
{"loss": 0.39595305919647217, "acc": 0.8349608778953552, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3276945650577545, "acc": 0.8648375868797302, "length": 3.999952793121338, "mode": "train", "epoch": 38}
{"loss": 0.4069899916648865, "acc": 0.8347655534744263, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.32432201504707336, "acc": 0.8660823106765747, "length": 3.9995365142822266, "mode": "train", "epoch": 39}
{"loss": 0.4009423553943634, "acc": 0.8351563215255737, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.32412558794021606, "acc": 0.8635134696960449, "length": 3.9978950023651123, "mode": "train", "epoch": 40}
{"loss": 0.40016305446624756, "acc": 0.8388671875, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.32297664880752563, "acc": 0.8632298707962036, "length": 3.999995708465576, "mode": "train", "epoch": 41}
{"loss": 0.4270600974559784, "acc": 0.8306640982627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.33147796988487244, "acc": 0.8634364008903503, "length": 3.9976260662078857, "mode": "train", "epoch": 42}
{"loss": 0.4042482376098633, "acc": 0.8253906965255737, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.32062333822250366, "acc": 0.8664945363998413, "length": 3.9991934299468994, "mode": "train", "epoch": 43}
{"loss": 0.39156007766723633, "acc": 0.8392577171325684, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.31463438272476196, "acc": 0.8679373860359192, "length": 3.9999327659606934, "mode": "train", "epoch": 44}
{"loss": 0.41103029251098633, "acc": 0.8345703482627869, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3162052631378174, "acc": 0.8671630620956421, "length": 3.999986410140991, "mode": "train", "epoch": 45}
{"loss": 0.40596359968185425, "acc": 0.8283203840255737, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.32241788506507874, "acc": 0.8651361465454102, "length": 3.999937057495117, "mode": "train", "epoch": 46}
{"loss": 0.40244758129119873, "acc": 0.8335937857627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3113015294075012, "acc": 0.8698447942733765, "length": 3.9999947547912598, "mode": "train", "epoch": 47}
{"loss": 0.40062397718429565, "acc": 0.8341797590255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3134215176105499, "acc": 0.8691297173500061, "length": 3.9999473094940186, "mode": "train", "epoch": 48}
{"loss": 0.41539323329925537, "acc": 0.8281249403953552, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.31202855706214905, "acc": 0.8698446750640869, "length": 3.9999680519104004, "mode": "train", "epoch": 49}
{"loss": 0.42282000184059143, "acc": 0.8265624642372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3074094355106354, "acc": 0.8700822591781616, "length": 3.9999282360076904, "mode": "train", "epoch": 50}
{"loss": 0.40570980310440063, "acc": 0.8343749046325684, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3074541389942169, "acc": 0.8740162253379822, "length": 3.999983310699463, "mode": "train", "epoch": 51}
{"loss": 0.41411224007606506, "acc": 0.8335937857627869, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3024534285068512, "acc": 0.8744332790374756, "length": 3.9999682903289795, "mode": "train", "epoch": 52}
{"loss": 0.40656226873397827, "acc": 0.8394532203674316, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.30537649989128113, "acc": 0.8722878098487854, "length": 3.9999685287475586, "mode": "train", "epoch": 53}
{"loss": 0.40570420026779175, "acc": 0.8384765982627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.30946871638298035, "acc": 0.874849796295166, "length": 3.999863862991333, "mode": "train", "epoch": 54}
{"loss": 0.42099058628082275, "acc": 0.8277344107627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.3285577893257141, "acc": 0.8672821521759033, "length": 3.9999747276306152, "mode": "train", "epoch": 55}
{"loss": 0.4114445447921753, "acc": 0.8359375596046448, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.32006168365478516, "acc": 0.8685919642448425, "length": 3.9998230934143066, "mode": "train", "epoch": 56}
{"loss": 0.40551090240478516, "acc": 0.8330078721046448, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.30460095405578613, "acc": 0.8725844621658325, "length": 3.9996917247772217, "mode": "train", "epoch": 57}
{"loss": 0.4090616703033447, "acc": 0.8408203721046448, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3051612973213196, "acc": 0.8770529627799988, "length": 3.9999265670776367, "mode": "train", "epoch": 58}
{"loss": 0.39274442195892334, "acc": 0.8445311784744263, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.30353349447250366, "acc": 0.8767685890197754, "length": 3.997753381729126, "mode": "train", "epoch": 59}
{"loss": 0.4067053198814392, "acc": 0.8337889909744263, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.30208903551101685, "acc": 0.8792008757591248, "length": 3.9999449253082275, "mode": "train", "epoch": 60}
{"loss": 0.40998372435569763, "acc": 0.833984375, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.80078125
